provd/go.mod

```diff
@@ -0,0 +1,42 @@
+module github.com/canonical/ubuntu-desktop-provision/provd
+
+go 1.21.3
+
+require (
+	github.com/coreos/go-systemd v0.0.0-20191104093116-d3cd4ed1dbcf
+	github.com/sirupsen/logrus v1.9.3
```

Comment:
I think you should run `go mod tidy` to remove the logrus dependency now that it’s not used.